### DISCUSSION TITLE
Cronjob while restricting access with htaccess

### DISCUSSION THREAD

**Discussion by jarrah791** [wavelog/wavelog#1137] [Q&A]
I've hosted Wavelog on a shared hosting. I want to restrict access to the whole installation by "AuthType Basic" in the htaccess. Of course this interferes with the cron master which will stop working. How can I use restricted access in combination with the cron master?

**Comment by phl0** [comment]
See the --user and --password for wget options:

https://www.gnu.org/software/wget/manual/wget.html
